packages/docusaurus-plugin-content-blog/src/blogFrontMatter.ts

```diff
@@ -27,14 +27,17 @@ export type BlogPostFrontMatter = {
   author_title?: string;
   author_url?: string;
   author_image_url?: string;
+  author_key?: string;
```

Comment:
For multi-author, should we use `author_keys` instead? 

I think `author_key` is fine and could accept `string | string[]`

Reply:
I'd prefer generic keys to creating new keys for things that are programmatically the same. Still I would probably allow `author_keys` for multiple authors, but use Joi to normalize that to `author_key`